tests/cephfs/test_io.py

```diff
@@ -0,0 +1,26 @@
+import traceback
+
+from tests.cephfs.cephfs_utilsV1 import FsUtils
+from tests.io.fs_io import fs_io
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    try:
+        log.info(f"MetaData Information {log.metadata} in {__name__}")
+        config = kw.get("config")
+        build = config.get("build", config.get("rhbuild"))
+        clients = ceph_cluster.get_ceph_objects("client")
+        if config.get("cephfs"):
+            fs_util = FsUtils(ceph_cluster)
+            fs_util.prepare_clients(clients, build)
+            fs_util.auth_list(clients)
+            fs_io_obj = fs_io(client=clients[0], config=config.get("cephfs"))
+            fs_io_obj.run_fs_io()
+        return 0
+    except Exception as e:
+        log.info(e)
+        log.info(traceback.format_exc())
```

Comment:
Putting `traceback` stack will provide lot data but to understand it will consume some time. Instead I suggest to raise specific exception in `tests/io/fs_io.py` and catch it here